### PR TITLE
Issue 4815: Silently ignoring trailing @ at follow requests

### PR DESCRIPTION
**Pull request by annando**
fixes https://github.com/friendica/friendica/issues/4815

### CODE REVIEW

**Comment by AlfredSK** [comment]
I'm not sure. Is 'trailing' the correct English term? Shouldn't it be 'preceding @'?

File: mod/follow.php [hunk 1]
```diff
@@ -60,7 +60,9 @@ function follow_content(App $a)
 	}
 
 	$uid = local_user();
-	$url = Strings::escapeTags(trim(defaults($_REQUEST, 'url', '')));
+
+	// Issue 4815: Silently removing a trailing @
```
Comment (MrPetovan) [inline mod/follow.php]: trailing = right
prefixing = left

Comment (annando) [reply mod/follow.php]: You and @AlfredSK are two nitpickers ;-)

Comment (AlfredSK) [reply mod/follow.php]: Haha. :-)

**Pull request merged**